package.json

```diff
@@ -48,6 +48,7 @@
     "babel-plugin-transform-exponentiation-operator": "6.24.1",
     "babel-preset-es2015-node": "6.1.1",
     "babel-preset-es2015-rollup": "3.0.0",
+    "babel-preset-stage-3": "^6.24.1",
```

Comment:
this should be an exact version. i must not have an `.npmrc` file that specifies that. mind adding one as part of this PR?